dagobah/daemon/api.py

```diff
@@ -32,7 +32,7 @@ def get_job():
     if not job:
         abort(400)
 
-    return job._serialize()
+    return job._serialize(use_snapshot='use_snapshot' in args)
```

Comment:
that's definitely a bug, kinda embarassing actually haha.

effectively it doesnt make a difference *at the moment* because the front end only uses true right now, but very good catch